docs/api-reference/map.md

```diff
@@ -493,14 +493,12 @@ If `reuseMaps` is set to `true`, when a map component is unmounted, the underlyi
 
 Note that since some map options cannot be modified after initialization, when reusing maps, only the reactive props and `initialViewState` of the new component are respected.
 
-#### `RTLTextPlugin`: string | false {#rtltextplugin}
+#### `RTLTextPlugin`: object {#rtltextplugin}

```

Comment:
```suggestion
#### `RTLTextPlugin`: object {#rtltextplugin}

Default: `undefined`
```

I think we should keep the pattern to make the default explicit which in this case clarifies that this is now opt in.